lms/static/scripts/frontend_apps/components/JSTORPicker.js

```diff
@@ -211,6 +217,36 @@ export default function JSTORPicker({ onCancel, onSelectURL }) {
             </div>
           )}
 
+          {metadata.data && (
```

Comment:
The JSX here could have been appended to the preceding `metadata.data &&` section, but I felt that separating it out made it easier to scan and understand as a separate "feature."